### PR TITLE
Conversion of object_gcr_x notebooks to Run2.2i

### PR DESCRIPTION
**Pull request by JoanneBogart**
Start with object_gcr_1_intro; hopefully will get to one or two more

### CODE REVIEW

**Comment by JoanneBogart** [comment]
I'm part way through the intro notebook.  Issues to be dealt with include
* appropriate update for this text in the **Applying filters and cuts** section: 

> Admittedly, this plot is a little underwhelming, these quality cuts only remove a very small number of objects. This is due in part to the fact that Run 1.2i ran from imSim outputs with essentially perfect Instrument Signature Removal (ISR). Also, Run 1.2i coadds only have a limited number of exposures, but defects will add up as the coadds get deeper.

The part about the plot being underwhelming is still true in my modified version.  I don't know if the statement about ImSim is still true.  And the coadds in Run 2.2i_dr3 are already somewhat deeper.
* lack of old hsc catalog.   Is that gone forever?  If so, is there something else we can use for comparison?

**Comment by JoanneBogart** [comment]
The first two object_gcr_..  notebooks have been converted to use Run 2.2i_dr3.  Since the hsc-pdr1-xmm catalog is no longer available to query, results from old renderings of these notebooks have been inserted.
For object_gcr_2 I used tract 3081, which had the greatest depth of all I looked at, but this is a debatable choice.  One of the depleted tracts would show more dramatic differences with the HSC DR1 plots.

**Comment by JoanneBogart** [comment]
For object_gcr_3 I needed an appropriate depth map so I wrote out the one calculated in object_gcr_2 (saved in `assets` subdirectory) and read it in.
The plots comparing high and low blendedness for skew and kurtosis look weird.
I temporarily uncommented some of the latter section (meant as exercises) to see if I could get it to run.

File: tutorials/object_gcr_2_lensing_cuts.ipynb [hunk 1]
```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# DC2 Object Catalog Run2.1i GCR tutorial -- Part II: Lensing Cuts\n",
+    "# DC2 Object Catalog Run2.2i GCR tutorial -- Part II: Lensing Cuts\n",
     "\n",
     "\n",
     "Owners: **Francois Lanusse [@EiffL](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@EiffL), Javier Sanchez [@fjaviersanchez](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@fjaviersanchez)**  \n",
-    "Last Verified to Run: **2018-11-19** (by @yymao)\n",
+    "Last Verified to Run: **2020-07-02** (by @jrbogart)\n",
```
[the remark this answers is not in the document]
Comment (fjaviersanchez) [reply tutorials/object_gcr_2_lensing_cuts.ipynb:11]: I would also change the `import lsst.afw.geom as afwGeom` to `import lsst.geom as afwGeom` (there are a bunch of deprecation warnings that go away just by doing this.

File: tutorials/object_gcr_3_challenges.ipynb [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "# DC2 Object Catalog Run2.1i GCR tutorial -- Part III: Guided Challenges\n",
```
Comment (fjaviersanchez) [inline tutorials/object_gcr_3_challenges.ipynb:7]: 2.1i -> 2.2i

File: tutorials/object_gcr_3_challenges.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
...
     "\n",
     "Owners: **Javier Sanchez [@fjaviersanchez](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@fjaviersanchez), Francois Lanusse [@EiffL](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@EiffL)**  \n",
-    "Last Run: **2018-11-19** (by @yymao)\n",
+    "Last Run: **2020-07-02** (by @jrbogart)\n",
     "\n",
     "This notebook is the third of four in the Run2.1i object catalog with GCR series ([Part I](object_gcr_1_intro.ipynb), [Part II](object_gcr_2_lensing_cuts.ipynb)). Here, we propose some challenges for you as science use cases of the object catalogs. We will provide a solution here but you are encouraged to create your own!\n",
```
Comment (fjaviersanchez) [inline tutorials/object_gcr_3_challenges.ipynb:12]: 2.1i -> 2.2i

**Comment by JoanneBogart** [comment]
Thanks, @fjaviersanchez !  I'll merge.

**Pull request merged**